src/components/ViewPort/ViewPort.tsx

```diff
@@ -2,10 +2,24 @@ import * as React from 'react'
 import { Props } from './ViewPort.types'
 import Preview from 'components/Preview'
 import './ViewPort.css'
+import Icon from 'components/Icon'
 
 export default class ViewPort extends React.PureComponent<Props> {
+  handleClose = () => {
+    const { onTogglePreview } = this.props
+    onTogglePreview(false)
```

Comment:
Is this `false` necessary? I think the container does not receive the value and hardcodes `false`

Maybe we can change the name on the container to `onClosePreview`?

Reply:
It's because the type of `onTogglePreview` is `typeof togglePreview`:

```
export type Props = {
  isPreviewing: boolean
  onTogglePreview: typeof togglePreview
}
```

That's the pattern we've been using on this project from what I saw (always typing callbacks with the `typeof` of some action)

I could add a default value of `false` to `togglePreview` so we don't have to pass that `false` and then I can rename it to `onClosePreview`

Reply:
Makes sense!, wasn't sure looking at the code but I thought that was the reason Typescript didn't yell about the `false`

I think adding a default is an option, but maybe we could make an exception here and use something like:

`onClosePreview: () => ReturnType<typeof togglePreview>`

for the type. What do you think?

Reply:
done ✅